Frameworks/UIKit/UIFontDescriptor.mm

```diff
@@ -181,6 +183,20 @@ - (NSArray*)matchingFontDescriptorsWithMandatoryKeys:(NSSet*)mandatoryKeys {
 /**
  @Status Interoperable
 */
+- (CGAffineTransform)matrix {
+    NSData* data = [self objectForKey:UIFontDescriptorMatrixAttribute];
+    if (data) {
+        CGAffineTransform ret;
+        [data getBytes:reinterpret_cast<byte*>(&ret) length:sizeof(CGAffineTransform)];
```

Comment:
oops

---
In reply to: [80597857](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80597857) [](ancestors = 80597857)